### ISSUE TITLE
Upload Gift Card page not working?

### ISSUE THREAD

**Issue by shadowbrush** [baragaun/first-spark-app#189]
I picked a brand and filled in the form, then hit the "Submit" button. But nothing happened. Should it work?

**Comment by jahanvi-baragaun** [comment]
There is no backend integration on submit. I think there is not api ready for this. 
On enter from keyboard user can see default alert dialog (on click event was missing).

I have added onclick event and I have made changes to show customised alert dialog on submit (which looks like this)

<img width="367" height="649" alt="Image" src="https://github.com/user-attachments/assets/4584a7f2-624d-4e20-80d5-3b34b64c2abb" />

**Comment by shadowbrush** [comment]
Got it, thanks, Jahanvi. @RaghvindYadav Let's plan to add this to the API
